libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -44,6 +44,11 @@ using namespace solidity;
 using namespace solidity::yul;
 using namespace solidity::util;
 
+langutil::SourceLocation solidity::yul::extractSourceLocationFromDebugData(shared_ptr<DebugData const> const& _debugData)
```

Comment:
Why do we need this and not just use `locationOf()` in this file?

Reply:
I was getting weird error when tried using `locationOf()` in this file. I wanted to revisit and try solving it once more